Sources/CombineWamp/Model/WampError.swift

```diff
@@ -0,0 +1,63 @@
+public struct WampError {
```

Comment:
Same

Reply:
(Ok, I might miss something here. If this is just the model layer, it should be ok)